### PR TITLE
Initial Bookings Controller & Views (Work in Progress)

### PR DESCRIPTION
**Pull request by mattysplash**

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  def destroy
+    @booking = Booking.find(params[:id])
+    @booking.destroy
+    redirect_to bookings_path, notice: 'Booking was successfully destroyed.'
+  end
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:start, :end, :price, :description, :space_id, :user_id, :confirmed)
```
Comment (tarakoomen) [inline app/controllers/bookings_controller.rb]: Quick comment here: we don't want to be permitting all these variables when the user creates a new booking. I'd allow start and end only.

File: app/views/bookings/_booking.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<div>
+  <p>
+    <strong>Name:</strong>
```
Comment (tarakoomen) [inline app/views/bookings/_booking.html.erb]: Name can be changed to Start date

**Review by tarakoomen — COMMENTED** [review]
You've done a lot! Awesome work. Just a couple small remarks in this review. I think I'm onto the error but let's talk about it tmrw. Happy for you to merge

**Comment by mattysplash** [comment]
Thanks Tara good catch!

I closed this request and an updated request will follow

**Pull request closed without merging**